app/javascript/topics/components/TopicsShow__PostShow.res

```diff
@@ -106,7 +130,9 @@ let optionsDropdown = (
         onClick={_ => archivePost(isFirstPost, post |> Post.id, archivePostCB)}
         className="flex w-full px-3 py-2 font-semibold items-center text-gray-700 whitespace-no-wrap">
         <FaIcon classes="fas fa-trash-alt fa-fw text-base" />
-        <span className="ml-2"> {"Delete " ++ postTypeString |> str} </span>
+        <span className="ml-2">
+          {(isFirstPost ? t("delete_post_string") : t("delete_reply_string")) |> str}
```

Comment:
Should this be `delete_topic_string` string instead of `delete_post_string`. The button currently reads `Delete Post`, when it should really read `Delete Topic`.